.travis.yml

```diff
@@ -10,18 +10,8 @@ python:
 
 env:
   # NOTE: keep this in sync with envlist in tox.ini for tox-travis.
-  - PIP=8.1.1
-  - PIP=9.0.1
-  - PIP=9.0.3
-  - PIP=10.0.1
-  - PIP=18.0
-  - PIP=19.0.3
-  - PIP=19.1
-  - PIP=19.2.3
-  - PIP=19.3
-  - PIP=20.0
+  # TODO: add PIP=20.0 after pip==20.1 being released
   - PIP=latest
-  - PIP=master
```

Comment:
I think it's a waste of time if it would be failing almost every time (as it does now for the instance). We have a cron job for master, this could be enough. But I don't insist. What do you think?

Reply:
I didn't notice the cron job, I'm totaly ok with that instead of running it on every MR 👍 .